src/lib/organisms/Table/Row/col.js

```diff
@@ -97,7 +104,7 @@ function Col({
             >
                 <SkeletonLoader height={20} isBusy={guidRef.current && promiseValue === guidRef.current}>
                     {copyable && value && (
-                        <Tooltip title={copyTooltipText}>
+                        <Tooltip title={isCopied ? copiedTooltipText : copyTooltipText}>
                             <Icon
```

Comment:
I suggest improving UX using the checkmark icon when the user clicks the copy icon.
It would be best if you had changed the icon type conditionally like the `copiedTooltipText` value. As a checkmark icon, I suggest using this one `bc-icon-checkbox-checked`.

https://github.com/softconstruct/gene-ui-components/assets/150047343/453d7ba7-a3ee-467e-8815-645c61ca4344